### PR TITLE
fix blocking discovery client in DiscoveryClientServiceInstanceListSupplier

### PR DESCRIPTION
**Pull request by rodbate**

### CODE REVIEW

File: spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/loadbalancer/core/DiscoveryClientServiceInstanceListSupplier.java [hunk 1]
```diff
@@ -39,6 +40,7 @@
  * @author Spencer Gibb
  * @author Olga Maciaszek-Sharma
  * @author Tim Ysewyn
+ * @author rodbate
```
Comment (spencergibb) [inline spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/loadbalancer/core/DiscoveryClientServiceInstanceListSupplier.java:43]: We don't use usernames in author tags.

**Review by spencergibb — APPROVED** [review]

File: spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/loadbalancer/core/DiscoveryClientServiceInstanceListSupplier.java (hunk 1 above)
Comment (rodbate) [reply spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/loadbalancer/core/DiscoveryClientServiceInstanceListSupplier.java]: GOT IT

**Comment by OlgaMaciaszek** [comment]
@rodbate Thanks for submitting the fix. Since this code is also used in `Hoxton` release train, I'm going to cherry-pick it to `2.2.x` in order to get it into the upcoming release.

**Comment by OlgaMaciaszek** [comment]
Closing with 84fb8550c776aaab48e943f24a1481041d8528f1.

**Pull request closed without merging**